### ISSUE TITLE
执行emqx_ctl status 报错exception, no match of right hand side value false

### ISSUE THREAD

**Issue by sunnyinchina** [emqx/emqx#7278]
开源版本4.3.12执行emqx_ctl status 报错exception, no match of right hand side value false

![image](https://user-images.githubusercontent.com/46027205/157816561-dbd31775-f7e7-4c41-9c5e-3363eade5b9a.png)

**Comment by inkydragon** [comment]
一样的问题。不过我尝试了 #7682 中的建议。
先 `emqx console` 再次尝试启动服务，显示服务已启动。
```
>emqx console
emqx is started!
```

然后 `emqx_ctl status` 就正常了
```
>emqx_ctl status
Node 'emqx@127.0.0.1' 4.4.3 is started
```

**Comment by zmstone** [comment]
Sorry for the late response.
I believe this has been fixed by this PR:
https://github.com/emqx/emqx/pull/5385
The fix was release in v4.3.14
